### PR TITLE
[CF/COL] Prosthetics Clinic, and 5 new Prosthetics

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds 5 new prosthetics to CF/COL.

An arm that blows up when it gets disabled
An extra arm that lowers your movespeed (also in double!)
And a mantis blade implant in red and black
A chainsword that deals multihits
A rat Zippy arm (mostly a joke, but isn't that bad)

All of these have big downsides, and are balanced around that.
All future prosthetics should have significant downsides.

## Why It's Good For The Game

Adds one extra level of depth to prosthetics.

## Changelog
:cl:
add: Added new prosthetics
add: Added prosthetics surgeon to CF/COL
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_items/prosthetics/chainsword.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/obj/item/organ/cyberimp/arm/chainsword
+	name = "chainsword implant"
+	desc = "A chainsword that can be retracted into your arm.."
+	contents = newlist(/obj/item/ego_weapon/city/handchainsword)
+	syndicate_implant = TRUE
+	//Doesn't need a nerf, the gun's kinda shit lmao
+
+/obj/item/organ/cyberimp/arm/chainsword/l
+	zone = BODY_ZONE_L_ARM
+	syndicate_implant = TRUE
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/chainsword.dm:10]: Already defined by /obj/item/organ/cyberimp/arm/chainsword, no need to re-define
```suggestion
```

File: ModularTegustation/tegu_items/prosthetics/mantis.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/obj/item/organ/cyberimp/arm/mantis
+	name = "Mantis blade implants"
+	desc = "Mantis blades designed by some sicko in district."
+	contents = newlist(/obj/item/ego_weapon/city/mantis)
+	syndicate_implant = TRUE
+
+/obj/item/organ/cyberimp/arm/mantis/l
+	zone = BODY_ZONE_L_ARM
+	syndicate_implant = TRUE
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/mantis.dm:9]: re-defined again
```suggestion
```

**Comment by Tokanova** [comment]
hows dis goin

File: ModularTegustation/tegu_items/prosthetics/extrarm.dm [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+/obj/item/mechandrites
+	name = "\improper Mechandrite Implanter"
+	desc = "An implanter for mechandrites, allowing a follower of the Omnissiah to gain newly found dexterity and handiness"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "autoimplanter"
+	var/uses = 1
+
+/obj/item/mechandrites/attack_self(mob/user)
+	if(!uses)
+		to_chat(user, span_notice("[src] has already been used. The tools are dull and won't reactivate.")
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extrarm.dm:27]:
```suggestion
		to_chat(user, span_notice("[src] has already been used. The tools are dull and won't reactivate."))
```

File: ModularTegustation/tegu_items/prosthetics/extrarm.dm [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+		return
+	var/limbs = user.held_items.len
+	if(limbs >= 4)
+		to_chat(user, span_alert("You already have an extra arm.")
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extrarm.dm:31]:
```suggestion
		to_chat(user, span_alert("You already have an extra arm."))
```

File: ModularTegustation/tegu_items/prosthetics/extrarm.dm [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+		return
+	user.change_number_of_hands(limbs+1)
+	user.visible_message(span_notice("[user] presses a button on [src], and you hear a short mechanical noise.", span_notice(">You feel a sharp sting as [src] plunges into your body.")
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extrarm.dm:34]:
```suggestion
	user.visible_message(span_notice("[user] presses a button on [src], and you hear a short mechanical noise."), span_notice("You feel a sharp sting as [src] plunges into your body."))
```

File: ModularTegustation/tegu_items/prosthetics/extra_arm_implant.dm [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+		to_chat(user, span_alert("[src] has already been used. The tools are dull and won't reactivate."))
+		return
+	if(user in GLOB.threearmed_fucks)	//not letting people get 6 arms
+		to_chat(user, span_alert("You already have an extra arm."))
+		return
+	var/limbs = user.held_items.len
+	user.change_number_of_hands(limbs+1)
+	GLOB.threearmed_fucks += user
+	user.visible_message(span_alert("[user] presses a button on [src], and you hear a short mechanical noise."), span_alert("You feel a sharp sting as [src] plunges into your body."))
+	to_chat(user, "Your extra arm whirrs with life"))
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extra_arm_implant.dm:24]:
```suggestion
	to_chat(user, span_notice("Your extra arm whirrs with life"))
```

File: ModularTegustation/tegu_items/prosthetics/extra_arm_implant.dm [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+		to_chat(user, span_alert("[src] has already been used. The tools are dull and won't reactivate."))
+		return
+	if(user in GLOB.threearmed_fucks)	//not letting people get 6 arms
+		to_chat(user, span_alert("You already have an extra arm."))
+		return
+	var/limbs = user.held_items.len
+	user.change_number_of_hands(limbs+2)
+	GLOB.threearmed_fucks += user
+	user.visible_message(span_notice("[user] presses a button on [src], and you hear a short mechanical noise."), span_alert("You feel a sharp sting as [src] plunges into your body."))
+	to_chat(user, "Your extra arm whirrs with life"))
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extra_arm_implant.dm:64]:
```suggestion
	to_chat(user, span_notice("Your extra arm whirrs with life"))
```

File: ModularTegustation/tegu_items/prosthetics/extrarm.dm [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+	if(!uses)
+		to_chat(user, span_notice("[src] has already been used. The tools are dull and won't reactivate."))
+		return
+	var/limbs = user.held_items.len
+	if(limbs >= 4)
+		to_chat(user, span_alert("You already have an extra arm."))
+		return
+	user.change_number_of_hands(limbs+1)
+	user.visible_message(span_notice("[user] presses a button on [src], and you hear a short mechanical noise."), span_notice("You feel a sharp sting as [src] plunges into your body."))
+	to_chat(user, "Your mechandrites whirr with life")
```
Comment (Gboster-0) [inline ModularTegustation/tegu_items/prosthetics/extrarm.dm:35]: also missing a span, i think this is the last one
```suggestion
	to_chat(user, span_notice("Your mechandrites whirr with life"))
```

**Review by Gboster-0 — APPROVED** [review]
Good as it gets, players enjoyed it, shipping it

**Pull request merged**